cmd/kyma/alpha/create/module/moduleconfig.go

```diff
@@ -28,6 +29,7 @@ const (
 	//taken from "github.com/open-component-model/ocm/resources/component-descriptor-v2-schema.yaml"
 	moduleNamePattern = "^[a-z][-a-z0-9]*([.][a-z][-a-z0-9]*)*[.][a-z]{2,}(/[a-z][-a-z0-9_]*([.][a-z][-a-z0-9_]*)*)+$"
 	moduleNameMaxLen  = 255
+	namespaceMaxLen   = 253
```

Comment:
Let's be consistent and extract the namespace regex pattern below as well into const?